breaking/entrypoint.sh

```diff
@@ -23,7 +23,32 @@ fi
 echo "flags: $flags"
 
 if [ -n "$flags" ]; then
-    oasdiff breaking "$base" "$revision" $flags
+    output=$(unbuffer oasdiff breaking "$base" "$revision" $flags)
```

Comment:
Do you think `unbuffer` is relevant?
I prefer the code simple as possible.

note: if we will decide that it is - do not forget to add it also to `diff` (currently you added it only in breaking and changelog).
reference: https://linux.die.net/man/1/unbuffer

Reply:
Unbuffer is only used here to circumvent the [`isPipedOutput` check](https://github.com/Tufin/oasdiff/blob/8a284be84de5bb7e2e3ce83a6fb6a602aedba0b4/checker/api_change.go#L66C11-L66C11).

This allows us to preserve the colored output in the GitHub actions log, and the nicely indented messages for the GitHub output variables.

![image](https://github.com/oasdiff/oasdiff-action/assets/692443/edba17de-780c-4e9b-8242-cbf7edc3f1cd)

![image](https://github.com/oasdiff/oasdiff-action/assets/692443/6083ce39-f9d8-4b88-87fe-2cd6f6ddd0d5)

That also explains why we don't need it for `diff`, because it produces the same output, whether run directly or piped into another command.

---

But yeah, it's accidental complexity that could be removed/refactored to create simpler code and a smaller container.